ExtraDry/ExtraDry.Blazor/Components/Dry/DryStaticFilter.razor.cs

```diff
@@ -0,0 +1,26 @@
+namespace ExtraDry.Blazor;
+public partial class DryStaticFilter : ComponentBase, IExtraDryComponent
+{
+    /// <inheritdoc cref="IExtraDryComponent.CssClass "/>
+    [Parameter]
+    public string CssClass { get; set; } = string.Empty;
+
+    /// <inheritdoc cref="IExtraDryComponent.UnmatchedAttributes" />
+    [Parameter(CaptureUnmatchedValues = true)]
+    public Dictionary<string, object>? UnmatchedAttributes { get; set; }
+
+    [Parameter]
+    public int? InitialHierarchyLevel { get; set; }
+
+    /// <inheritdoc cref="DryPageQueryView.PageQueryBuilder" />
+    [CascadingParameter]
+    public QueryBuilder? QueryBuilder { get; set; }
+
+    protected override Task OnInitializedAsync()
```

Comment:
The filter value might not exist at Initialize and could technically change during the lifecycle.  OnParametersSet might be a better lifecycle event for this.